docs/sources/enterprise/_index.md

```diff
@@ -66,6 +66,7 @@ Grafana Enterprise adds the following features:
 - [Custom branding]({{< relref "../setup-grafana/configure-grafana/configure-custom-branding/" >}}) to customize Grafana from the brand and logo to the footer links.
 - [Usage insights]({{< relref "usage-insights/" >}}) to understand how your Grafana instance is used.
 - [Vault integration]({{< relref "../setup-grafana/configure-security/configure-database-encryption/integrate-with-hashicorp-vault/" >}}) to manage your configuration or provisioning secrets with Vault.
+- [AWS Secrets Manager integration]({{< relref "aws-secrets-plugin/" >}}) to store datasource secrets in AWS Secrets Manager.
```

Comment:
```suggestion
- [AWS Secrets Manager integration]({{< relref "../setup-grafana/configure-security/configure-database-encryption/encrypt-secrets-using-aws-secrets-manager/" >}}) to store datasource secrets in AWS Secrets Manager.
```

See comment in aws-secrets-plugin.md about the expected location for these plugins.

Reply:
moved file and updated